<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
<comment> Begin CVS Header </comment>
<comment>   $Source: /Volumes/Home/Users/shoops/cvs/copasi_dev/copasi/UI/CQTSSAWidget.ui,v $ </comment>
<comment>   $Revision: 1.9 $ </comment>
<comment>   $Name:  $ </comment>
<comment>   $Author: shoops $ </comment>
<comment>   $Date: 2011/09/19 13:54:23 $ </comment>
<comment> End CVS Header </comment>

<comment> Copyright (C) 2011 by Pedro Mendes, Virginia Tech Intellectual </comment>
<comment> Properties, Inc., University of Heidelberg, and The University </comment>
<comment> of Manchester. </comment>
<comment> All rights reserved. </comment>

<comment> Copyright (C) 2008 by Pedro Mendes, Virginia Tech Intellectual </comment>
<comment> Properties, Inc., EML Research, gGmbH, University of Heidelberg, </comment>
<comment> and The University of Manchester. </comment>
<comment> All rights reserved. </comment>

 <comment> All rights reserved. </comment>
 <class>CQTSSAWidget</class>
 <widget class="TaskWidget" name="CQTSSAWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>364</width>
    <height>183</height>
   </rect>
  </property>
  <property name="caption" stdset="0">
   <string>Time Scale Separation Analysis</string>
  </property>
  <layout class="QVBoxLayout">
   <item>
    <layout class="QGridLayout">
     <item row="1" column="3">
      <widget class="QLineEdit" name="mpEditIntervals"/>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="mpEditIntervalSize"/>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="mpEditDuration"/>
     </item>
     <item row="1" column="2">
      <widget class="QLabel" name="mpLblIntervals">
       <property name="text">
        <string>Intervals</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
       <property name="wordWrap">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="mpLblDuration">
       <property name="text">
        <string>Duration</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
       <property name="wordWrap">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="mpLblIntervalSize">
       <property name="text">
        <string>Interval Size</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
       <property name="wordWrap">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="2" column="1" colspan="2">
      <widget class="QCheckBox" name="mpCheckSave">
       <property name="text">
        <string>Save Result in Memory</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="Line" name="line1">
     <property name="frameShape">
      <enum>QFrame::HLine</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Sunken</enum>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>TaskWidget</class>
   <extends>QWidget</extends>
   <header>TaskWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>mpEditDuration</tabstop>
  <tabstop>mpEditIntervalSize</tabstop>
  <tabstop>mpEditIntervals</tabstop>
  <tabstop>mpCheckSave</tabstop>
 </tabstops>
 <includes>
  <include location="local">TaskWidget.h</include>
  <include location="local">TaskWidget.h</include>
 </includes>
 <resources/>
 <connections>
  <connection>
   <sender>mpEditDuration</sender>
   <signal>editingFinished()</signal>
   <receiver>CQTSSAWidget</receiver>
   <slot>slotDuration()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>mpEditIntervalSize</sender>
   <signal>editingFinished()</signal>
   <receiver>CQTSSAWidget</receiver>
   <slot>slotIntervalSize()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>mpEditIntervals</sender>
   <signal>editingFinished()</signal>
   <receiver>CQTSSAWidget</receiver>
   <slot>slotIntervals()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
